lib/pdk/util/template_uri.rb

```diff
@@ -7,6 +7,7 @@ class TemplateURI
 
       PACKAGED_TEMPLATE_KEYWORD = 'pdk-default'.freeze
       DEPRECATED_TEMPLATE_URL = 'https://github.com/puppetlabs/pdk-module-template'.freeze
+      PDK_TEMPLATE_URL = 'https://github.com/puppetlabs/pdk-templates'.freeze
```

Comment:
Is this a semi-duplicate of   https://github.com/puppetlabs/pdk/blob/bd5e6cea2d528a758a6d93cf756accc100abd16a/lib/pdk/util/template_uri.rb#L133 ?

Reply:
if it is can we just `Addressable::URI.new(PDK_TEMPLATE_URL) ` instead?